tests/baselines/reference/extendedUnicodePlaneIdentifiers.js

```diff
@@ -2,9 +2,23 @@
 const 𝑚 = 4;
 const 𝑀 = 5;
 console.log(𝑀 + 𝑚); // 9
+
+// lower 8 bits look like 'a'
+const ၡ = 6;
+console.log(ၡ ** ၡ);
+
+// lower 8 bits aren't a valid unicode character
+const ဒ = 7;
```

Comment:
This only tests a single character, so you don't have a test that hits the loop body of `isIdentifierText`. Consider adding a test with multiple characters in the SMP or SIP, or combinations of BMP/SMP/SIP characters.

Reply:
Done~